tests/python/pants_test/util/test_retry.py

```diff
@@ -15,46 +15,46 @@ class RetryTest(unittest.TestCase):
   @mock.patch('time.sleep', autospec=True, spec_set=True)
   def test_retry_on_exception(self, mock_sleep):
     broken_func = mock.Mock()
-    broken_func.side_effect = IOError('broken')
+    broken_func.side_effect = OSError('broken')
```

Comment:
Python 3.3 consolidated multiple OS related errors into just `OSError`. It will still work to have it as `IOError`, but is confusing in logs because Py3 converts `IOError -> OSError` behind-the-scenes.